kolibri/plugins/facility/assets/src/views/ManageClassPage/index.vue

```diff
@@ -234,11 +238,6 @@
         message: '{name1}, {name2}… (+{numRemaining, number})',
         context: 'DO NOT TRANSLATE\nCopy the source string.',
       },
-      noClassesExist: {
```

Comment:
I believe it would be more preferable to display this string when there are no classes available, instead of presenting an empty table :)